lib/Dependency.js

```diff
@@ -7,7 +7,7 @@ function Dependency() {
 }
 module.exports = Dependency;
 
-Dependency.prototype.isEqualResource = function( /* other */ ) {
```

Comment:
I tried enabling it, but it resulted in so many errors that I thought it was best to fix this later.